independent-projects/arc/tests/src/test/java/io/quarkus/arc/test/cdi/bcextensions/RegistrationTest.java

```diff
@@ -57,6 +68,14 @@ public void observers(ObserverInfo observer, Types types) {
                 observerQualifierCounter.addAndGet(observer.qualifiers().size());
             }
         }
+
+        @Registration(types = MyInterceptor.class)
+        public void interceptors(BeanInfo interceptor, Messages msg) {
```

Comment:
Do we have a test case where the param is `InterceptorInfo` instead?

Reply:
I don't think we do, no.